src/test/feature-implicit-init.spec.ts

```diff
@@ -45,4 +46,12 @@ describe('feature-send', () => {
         expect(await contract.getGetCounter()).toBe(2n);
         expect(tracker.collect()).toMatchSnapshot();
     });
+
+    it('should not compile with uninitialized storage fields', async () => {
+        const result = await run({
+            configPath: __dirname + '/test-tact.config.json',
+            projectNames: ['implicit-init-2'],
+        });
+        expect(result).toBe(false);
```

Comment:
We should test against the error the user will see, not just to check if it does not compile. The point is to ensure the error message is understandable in this case.